playbooks/dependencies-fedora.yaml

```diff
@@ -39,6 +39,40 @@
       - udisks2
     use: "{{ 'dnf' if zuul.attempts > 1 else 'auto' }}"
 
+- name: Dump current /etc/subuid contents
+  become: true
+  command: cat /etc/subuid
```

Comment:
Nitpick: do we really need to run `cat /etc/subuid` as root?  The file is world-readable.